dbt/include/spark/macros/materializations/incremental/incremental.sql

```diff
@@ -1,46 +1,64 @@
 {% materialization incremental, adapter='spark' -%}
-  
   {#-- Validate early so we don't run SQL if the file_format + strategy combo is invalid --#}
   {%- set raw_file_format = config.get('file_format', default='parquet') -%}
   {%- set raw_strategy = config.get('incremental_strategy', default='append') -%}
-  
   {%- set file_format = dbt_spark_validate_get_file_format(raw_file_format) -%}
   {%- set strategy = dbt_spark_validate_get_incremental_strategy(raw_strategy, file_format) -%}
-  
-  {%- set unique_key = config.get('unique_key', none) -%}
-  {%- set partition_by = config.get('partition_by', none) -%}
 
-  {%- set full_refresh_mode = (should_full_refresh()) -%}
-  
-  {% set on_schema_change = incremental_validate_on_schema_change(config.get('on_schema_change'), default='ignore') %}
-
-  {% set target_relation = this %}
-  {% set existing_relation = load_relation(this) %}
-  {% set tmp_relation = make_temp_relation(this) %}
+  {#-- Set vars --#}
+  {%- set unique_key = config.get('unique_key', none) -%}
+  {%- set partition_by = config.get('partition_by', none) -%}  
+  {%- set language = config.get('language') -%}
```

Comment:
Also the implementations seems to be in `dbt-databricks`? And raise an error in `dbt-spark`?